relay-statsd/src/lib.rs

```diff
@@ -247,7 +250,24 @@ pub fn init<A: ToSocketAddrs>(
     let socket = UdpSocket::bind("0.0.0.0:0").unwrap();
     socket.set_nonblocking(true).unwrap();
 
-    let statsd_client = if buffering {
+    let statsd_client = if aggregating {
+        let host = host.to_socket_addrs().unwrap().next().unwrap();
+        let statsdproxy_sink = StatsdProxyMetricSink::new(move || {
+            let next_step = statsdproxy::middleware::upstream::Upstream::new(host)
+                .expect("failed to create statsdproxy metric sink");
+            statsdproxy::middleware::aggregate::AggregateMetrics::new(
+                AggregateMetricsConfig {
+                    aggregate_gauges: true,
+                    aggregate_counters: true,
+                    flush_interval: 1,
+                    flush_offset: 0,
+                    max_map_size: None,
```

Comment:
Should these be configurable?

Reply:
they should probably not have been options to begin with tbh